test/cookbooks/app_test/recipes/app2.rb

```diff
@@ -1,36 +1,32 @@
-include_recipe 'osl-mysql::server'
+node.force_override['osl-mysql']['enable_percona_client'] = false
 
 replicant_dbcreds = data_bag_item('replicant_redmine', 'mysql_creds')
 replicant_dbcreds['db_hostname'] = '172.%'
 
-percona_mysql_user replicant_dbcreds['db_user'] do
-  host replicant_dbcreds['db_hostname']
+osl_mysql_test replicant_dbcreds['db_db'] do
+  username replicant_dbcreds['db_user']
   password replicant_dbcreds['db_passwd']
-  ctrl_password ''
-  action :create
 end
 
-percona_mysql_database replicant_dbcreds['db_db'] do
-  password ''
-end
-
-percona_mysql_user replicant_dbcreds['db_user'] do
-  host replicant_dbcreds['db_hostname']
+# Modify the replicant user to allow for DB connections outside of localhost
+mariadb_user replicant_dbcreds['db_user'] do
+  ctrl_password 'osl_mysql_test'
   database_name replicant_dbcreds['db_db']
-  privileges ['ALL PRIVILEGES']
-  table '*'
   password replicant_dbcreds['db_passwd']
-  ctrl_password ''
+  host '%'
+  privileges [:all]
   action :grant
 end
 
+osl_firewall_port 'mysql'
```

Comment:
Hmm, it might be a good idea to add this to the `osl_mysql_test` resource.

Reply:
Implemented in the latest PR to `osl-mysql`.